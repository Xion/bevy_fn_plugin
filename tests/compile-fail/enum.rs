#[macro_use] extern crate bevy_fn_plugin;

#[bevy_plugin]
enum Enum {
    Foo,
    Bar,
}

fn main() {}
