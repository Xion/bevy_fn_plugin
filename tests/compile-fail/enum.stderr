error: expected `fn`
 --> tests/compile-fail/enum.rs:4:1
  |
4 | enum Enum {
  | ^^^^
